### PR TITLE
feat(arc-saas): adding new feature named identity feature

### LINKED ISSUE #85: (arc-saas):add IDP feature in Feature
**Describe the solution you'd like**

add IDP feature in Feature

### PR DESCRIPTION
**Pull request by Deepika516**
## Description
adding new feature named identity feature

Fixes #85

### CODE REVIEW

File: projects/saas-ui/src/app/shared/models/feature.model.ts [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 
 export class Features {
+  [x: string]: any;
   id?: string;
   name: string;
   description: string;
   key: string;
   type: 'boolean' | 'number' | 'string' | 'object';
-  defaultValue: any;
+  defaultValue: number;
```
Comment (yeshamavani) [inline projects/saas-ui/src/app/shared/models/feature.model.ts:11]: why is defaultValue type number 
it is string

File: projects/saas-ui/src/app/main/components/add-plan/add-plan.component.ts [hunk 2]
```diff
@@ -374,6 +374,13 @@ export class AddPlanComponent implements OnInit {
   getFeatures() {
     this.featureListService.getFeatures().subscribe(res => {
       this.featureOption = res;
+      // error handling
+      this.featureOption.forEach(feature => {
+        if (typeof feature.metadata === 'string') {
+          feature.metadata = feature.metadata.split(',');
+        }
+      });
+      console.log(this.featureOption);
```
Comment (yeshamavani) [inline projects/saas-ui/src/app/main/components/add-plan/add-plan.component.ts:383]: remove consoles

File: projects/saas-ui/src/app/main/components/add-plan/add-plan.component.html [hunk 3]
```diff
@@ -189,25 +189,63 @@ <h3>Plan Configuration</h3>
...
+                            {{ option }}
+                          </option>
+                        </select> -->
+                        <div class="select">
+                          <nb-select
+                            *ngIf="
+                              feature.metadata && feature.metadata.length > 0
+                            "
+                            class="dropdown-wrapper"
+                            placeholder="Select IDP"
```
Comment (yeshamavani) [inline projects/saas-ui/src/app/main/components/add-plan/add-plan.component.html:227]: when the handling is dynamic how can the placeholder be specific to Idp only

**Review by yeshamavani — CHANGES_REQUESTED** [review]

File: projects/saas-ui/src/app/shared/models/feature.model.ts (hunk 1 above)
Comment (Deepika516) [reply projects/saas-ui/src/app/shared/models/feature.model.ts]: done usme sb types hai default value mai , number bhi string bhi boolean bhi

File: projects/saas-ui/src/app/main/components/add-plan/add-plan.component.ts (hunk 2 above)
Comment (Deepika516) [reply projects/saas-ui/src/app/main/components/add-plan/add-plan.component.ts]: Done

File: projects/saas-ui/src/app/main/components/add-plan/add-plan.component.html (hunk 3 above)
Comment (Deepika516) [reply projects/saas-ui/src/app/main/components/add-plan/add-plan.component.html]: removed

**Review by yeshamavani — APPROVED** [review]

**Pull request merged**